website/index.html

```diff
@@ -900,6 +900,12 @@
           <img src="assets/images/sponsors/gcp_F19.svg" alt="GCP" class="img-fluid GCP-night">
         </a>
       </div>
+      <div class="col-sm-6">
+        <!-- Textron -->
+        <a href="https://cloud.google.com/" target="_blank">
+          <img src="assets/images/sponsors/textron_day.svg" alt="textron" class="img-fluid textron-night">
+        </a>
+      </div>
```

Comment:
The container row div for this should add the bootstrap class _justify-content-center_ so that the Textron logo is centered on the next line when it flex-wraps

Reply:
Yeah, I'd just make this change to make it look a bit cleaner.